### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by xeruf**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

- [ ]  issue was opened to discuss proposed changes before starting implementation. It is important do discuss changes before implementing them (Why should we add it? How should it work? How should it look? Where will it be? ...).
- [x]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [x]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [x]  app version number was not changed.
- [x]  all new code has tests to ensure against regressions.
- [x] `npm run lint` reports no offenses.
- [x] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.
- [x]  after PR is approved, all commits in it are [squashed](https://gitbetter.substack.com/p/how-to-squash-git-commits)

### CODE REVIEW

**Comment by hovancik** [comment]
I have the same structure on the website, so I don't wanna change it; there is reasoning behind it.

**Comment by xeruf** [comment]
I did not really change the structure, mostly wording issues. I can change that heading back if that is the issue.

File: README.md [hunk 1]
```diff
@@ -2,22 +2,18 @@
 
 <img src="stretchly_128x128.png" align="right" alt="Stretchly logo">
 
-> **The break time reminder app**
```
Comment (hovancik) [inline README.md:5]: I would like to keep this.

File: README.md [hunk 2]
```diff
@@ -2,22 +2,18 @@
...
-*Stretchly* is a cross-platform [Electron](https://www.electronjs.org/) app that reminds you to take breaks when working on your computer.
+*Stretchly* is a cross-platform [Electron](https://www.electronjs.org/) app that reminds you to take breaks while working at the computer.
 
 [![Become a Sponsor!](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&logo=GitHub&color=success)](https://github.com/sponsors/hovancik) [![Become a Patron!](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&logo=Patreon&color=success)](https://www.patreon.com/hovancik)
 
 ## Table of contents
 - [Install](#install--)
 - [Default behavior](#default-behavior)
 - [Preferences](#preferences)
-- [Advanced Preferences](#advanced-preferences)
```
Comment (hovancik) [inline README.md:15]: I would like to keep this.

File: README.md [hunk 3]
```diff
@@ -2,22 +2,18 @@
...
-- [Contributor Preferences](#contributor-preferences)
 - [Development](#development)
 - [Known issues](#known-issues)
 - [Contributors](#contributors)
-- [Humans and Tools](#humans-and-tools)
```
Comment (hovancik) [inline README.md:20]: I would like to keep this.

File: README.md [hunk 4]
```diff
@@ -430,14 +425,14 @@ Also see Github's list of [contributors](https://github.com/hovancik/stretchly/g
  - https://www.onhealth.com/content/1/eye_exercises_and_stretches
  - https://www.webmd.com/pain-management/exercises-carpal-tunnel-syndrome
 
-#### Sounds credits
```
Comment (hovancik) [inline README.md:433]: I would like to keep this.

File: README.md [hunk 5]
```diff
@@ -430,14 +425,14 @@ Also see Github's list of [contributors](https://github.com/hovancik/stretchly/g
...
+### Sounds
 Sounds used in this application are listed [here](http://freesound.org/people/hovancik/bookmarks/category/58865/).
 - `crystal glass` by [mlteenie](http://freesound.org/people/mlteenie/), available under the [Attribution License](http://creativecommons.org/licenses/by/3.0/).
 - `wind chime` by [GnoteSoundz](http://freesound.org/people/GnoteSoundz/), available under the [Creative Commons 0 License](http://creativecommons.org/publicdomain/zero/1.0/).
 - `tic toc` by [magundah14](http://freesound.org/people/magundah14/), available under the [Creative Commons 0 License](http://creativecommons.org/publicdomain/zero/1.0/).
 - `silence` by [parcodeisuoni](http://freesound.org/people/parcodeisuoni/), available under the [Attribution License](http://creativecommons.org/licenses/by/3.0/).
```
Comment (hovancik) [inline README.md:434]: I would like to keep this.

**Comment by hovancik** [comment]
Ok, I've commented in  codeblocks that I wanna keep.

File: README.md (hunk 2 above)
Comment (xeruf) [reply README.md:15]: It stays, I just made them subsections so it does not make much sense for it to show up in the ToC, no?
I feel like the ToC was a little overwhelming before.

Comment (hovancik) [reply README.md:15]: I link to them quit often in support issues/emails, so I prefer to have them

Comment (xeruf) [reply README.md:15]: You can still link to them, even if they don't appear in the ToC ;)

**Pull request closed without merging**